server/src/optima/project.py

```diff
@@ -324,6 +324,31 @@ def exportGraph():
     (dirname, basename) = os.path.split(path)
     return helpers.send_file(path, mimetype='application/vnd.openxmlformats-officedocument.spreadsheetml.sheet')
 
+"""
+saves All data as Excel files
+"""
+@project.route('/exportall', methods=['POST'])
+@login_required
+@report_exception()
+def exportAllGraphs():
+    from sim.makeworkbook import OptimaMultiSheetGraphTable
+    
+    data = json.loads(request.data)
+    project_name = request.headers['Project']
+    
```

Comment:
Can't we simply pass the data structure as parameter for OptimaMultiSheetGraphTable?